demo/variable_definitions/read_variable_definitions.ipynb

```diff
@@ -0,0 +1,375 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Variable Definitions API demo\n",
+    "\n",
+    "## Introduction\n",
+    "\n",
+    "Demonstrate a selection of the operations available on the Vardef internal API. This API is intended for creation and maintenance of _Variable Definitions_ by employees at Statistics Norway.\n",
+    "\n",
+    "## Get started\n",
+    "\n",
+    "This demo is intended to be run on [Dapla Lab](https://lab.dapla-test.ssb.no). Currently Vardef is not deployed to production, only the test environment, so it must be used from the Dapla Lav test environment (<https://lab.dapla-test.ssb.no>). Follow these instructions to get started.\n",
+    "\n",
+    "1. [Log in to Dapla Lab](https://manual.dapla.ssb.no/statistikkere/dapla-lab.html)\n",
+    "1. Start either the [Jupyter](https://manual.dapla.ssb.no/statistikkere/jupyter.html) or [VSCode](https://manual.dapla.ssb.no/statistikkere/vscode-python.html) service with [Data Access configured](https://manual.dapla.ssb.no/statistikkere/rstudio.html#buckets).\n",
+    "1. Clone this repo `git clone https://github.com/statisticsnorway/dapla-toolbelt-metadata.git`\n",
+    "1. Install the packages `poetry install; poetry shell`\n",
+    "1. Open this notebook\n",
+    "1. Choose the correct kernel\n",
+    "1. Run this notebook"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from typing import Any\n",
+    "\n",
+    "import requests\n",
+    "from rich import print_json\n",
```

Comment:
No matter what I do 'rich' always return 'ModuleNotFoundError '. I feel that 'print_json' is not very important for testing demo, a simple 'print()' is enough

Reply:
@tilen1976 I think you must have installed dependencies while on the main branch? I have had this failure, but it was only when I made the mistake of installing from the wrong branch.

Json printed with normal print is absolutely awful to read so I feel like this is important.